### ISSUE TITLE
Several twisted.conch.test.test_checkers.SSHProtocolCheckerTestCase test methods fail when PyCrypto is unavailable

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3709]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3709|
|Type|defect|
|Created|2009-03-30 14:29:22Z|
|Branch|https://github.com/twisted/twisted/tree/conch-tests-without-crypto-3709|

This can be seen on the without-modules builder:

http://buildbot.twistedmatrix.com/builders/py2.5-without-modules

Also, some of the test methods are misnamed (using underscores instead of camel case) and one is missing a docstring.

Attachments:

* [patch 3909.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/fee/fee882c1e9b3200f9ada43bc430571e0295d0ded/4b20919705e47abd7c01b77e7bd71e8226968521.diff) (3218 bytes) - added by christianmlong on 2009-04-01 04:50:57Z - Added skips, cleaned up test names

<details><summary>Searchable metadata</summary>

```
trac-id__3709 3709
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_conch_tests_without_crypto_3709 branches/conch-tests-without-crypto-3709
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1238423362000000 1238423362000000
changetime__1238705040000000 1238705040000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="christianmlong's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|christianmlong removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [26635]) Branching to 'conch-tests-without-crypto-3709'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [26636]) Apply patch to skip tests which cannot run without PyCrypto

refs [#3709](https://github.com/twisted/twisted/issues/3709)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [26637]) Merge conch-tests-without-crypto-3709

Author: christianmlong
Reviewer: exarkun
Fixes: [#3709](https://github.com/twisted/twisted/issues/3709)

Skip some more Conch tests which depend on PyCrypto when PyCrypto is not
installed.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
